scripts/error_codes.py

```diff
@@ -171,7 +171,8 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
     test_sub_dirs = [
         path.join("test", "libsolidity", "errorRecoveryTests"),
         path.join("test", "libsolidity", "smtCheckerTests"),
-        path.join("test", "libsolidity", "syntaxTests")
+        path.join("test", "libsolidity", "syntaxTests"),
+        path.join("test", "libyul", "yulSyntaxTests")
```

Comment:
Please also add `".yul"` to the list of extensions below:
```
    test_file_names = find_files(
        top_dir,
        test_sub_dirs,
        [".sol", ".yul"]
    )
```

Sorry for slow responses, I'm on Baltic see for vacations :).